src/code/z_prenmi.c

```diff
@@ -1,4 +1,5 @@
 #include "z_prenmi.h"
+#include "global.h"
```

Comment:
I'm used to seeing `#include "global.h"` as the first include which is common in code files. Is there any reason for one over the other (having it after other includes) or does it not really matter?

Reply:
Sounds good (there are also 4 more global.h put at the bottom in this PR)